### PR TITLE
Front end 2018

### PR DESCRIPTION
**Pull request by momeg**

### CODE REVIEW

**Comment by mehdibaha** [comment]
Great job on the PR! The commits are really messy though. It should go like this:

- Commit with added images, and changed image links in homepage
- Commit with only styling changes
- Commit with copy changes (copy == text)

File: app/templates/users/dashboard/sections/dashboard.html [hunk 1]
```diff
@@ -19,7 +19,7 @@ <h3 class="profile-username text-center">
             {% if not user.profile.first_name %}
               Profil incomplet
             </h3>
-            <p class="text-muted text-center">Veuillez remplir votre profil pour finaliser votre inscription.</p>
+            <p class="text-muted text-center">Veuillez remplir votre profil pour finaliser votre inscription et déposer votre CV pour tenter de gagner un Macbook Air le jour du salon. </p>
```
Comment (mehdibaha) [inline app/templates/users/dashboard/sections/dashboard.html]: Put a small link like `<a href="...">voir règlement</a>`

"*" are for billboards not websites

File: app/templates/users/master-class.html [hunk 2]
```diff
@@ -9,13 +9,13 @@
...
           </div>
-          <p class="heading-pre-suf text-center text-uppercase" data-selector="Coming Soon">15 Février, Amphi Emilie Du Châtelet - BMC</p>
+            <p class="heading-pre-suf text-center text-uppercase" data-selector="Coming Soon"><strong>Cet évènement est cloturé pour cette édition</strong></p>
           <h1 class="heading-text" data-selector="Coming Soon">{{ _("Master Class") }}</h1>
           <p class="sub-heading" data-selector="Coming Soon">Rendez-vous avec l'EM Lyon, l'IE Business School
             <br>et bien d'autres pour débattre sur les Masters Spécialisés !<br></p>
           <div class="home-download-btn">
             <a href="{{ url_for('users.dashboard', page='master_class') }}" class="btn btn-white btn-lg">
-              {{ 'Modifier ma participation' if current_user.is_authenticated else "S'inscrire à la Master Class" }}
+              {{ 'Indisponible' if current_user.is_authenticated else "S'inscrire à la Master Class" }}
```
Comment (mehdibaha) [inline app/templates/users/master-class.html]: Indisponible shouldn't redirect to dashboard, or was that your intention?

File: app/templates/users/start-up-your-future.html [hunk 3]
```diff
@@ -9,9 +9,9 @@
             <!-- REPLACE WITH YOUR LOGO -->
             <br>
           </div>
-          <p class="heading-pre-suf text-center text-uppercase" data-selector="Coming Soon">Le week-end pour entreprendre</p>
-          <h1 class="heading-text" data-selector="Coming Soon">Start-Up Your Future</h1>
-          <p class="sub-heading" data-selector="Coming Soon">{{ _("3, 4, 5 Février 2017") }}<br></p>
+            <p class="heading-pre-suf text-center text-uppercase" data-selector="Coming Soon"><strong>Cet évènement est cloturé pour cette édition</strong></p>
+          <h1 class="heading-text" data-selector="Coming Soon"></h1>
+          <p class="sub-heading" data-selector="Coming Soon"><br></p>
```
Comment (mehdibaha) [inline app/templates/users/start-up-your-future.html]: You forgot the "Indisponible" part...

File: app/templates/users/master-class.html [hunk 4]
```diff
@@ -65,8 +65,8 @@ <h3 class="title-text">{{ _("Une conférence interactive") }}</h3>
               </i>
             </div>
             <div class="media-body">
-              <h3 class="title-text">Un seul rendez-vous</h3>
-              <p>Mercredi 15 Février 2017 à 12h30, Amphi Emilie du Châtelet (Bibliothèque Marie Curie)
+              <h3 class="title-text">Fermé</h3>
+              <p>Cloturé pour cette édition
```
Comment (mehdibaha) [inline app/templates/users/master-class.html]: No reason to put this here: this is a past event, normal to have a passed date

File: app/templates/users/start-up-your-future.html [hunk 5]
```diff
@@ -9,9 +9,9 @@
             <!-- REPLACE WITH YOUR LOGO -->
             <br>
           </div>
-          <p class="heading-pre-suf text-center text-uppercase" data-selector="Coming Soon">Le week-end pour entreprendre</p>
-          <h1 class="heading-text" data-selector="Coming Soon">Start-Up Your Future</h1>
-          <p class="sub-heading" data-selector="Coming Soon">{{ _("3, 4, 5 Février 2017") }}<br></p>
+            <p class="heading-pre-suf text-center text-uppercase" data-selector="Coming Soon"><strong>Cet évènement est cloturé pour cette édition</strong></p>
```
Comment (mehdibaha) [inline app/templates/users/start-up-your-future.html]: No reason to remove the big title, only the date

File: app/templates/users/start-up-your-future.html [hunk 6]
```diff
@@ -9,9 +9,9 @@
...
+          <h1 class="heading-text" data-selector="Coming Soon"></h1>
```
Comment (mehdibaha) [inline app/templates/users/start-up-your-future.html]: Both these title are empty

File: app/users/views.py [hunk 7]
```diff
@@ -94,59 +95,64 @@ def signup():
             return redirect(url_for('users.signin'))
     return render_template('users/signup.html')
 
-#reset pass request , get email adress
+# reset pass request , get email adress
+
```
Comment (mehdibaha) [inline app/users/views.py]: is this whitespace for PEP8?

File: app/templates/users/forum-rhone-alpes.html [hunk 8]
```diff
@@ -102,7 +102,7 @@ <h2 class="heading-text">{{ _("Les entreprises présentes") }}</h2>
       <div class="row">
         <div class="col-md-12 text-center">
           <ul class="client-logo">
-            {% set companies = [ 'beijaflore', 'eiffage', 'loreal', 'mars', 'mbda', 'michelin', 'pg', 'reel', 'sncf', 'solutec', 'wavestone', 'groupeadf', 'groupehn', 'ge', 'homeserve' ] %}
+            {% set companies = [ 'ADDACTIS', 'AFD', 'ALUMNI_INSA_LYON', 'APLUS_CONSULTING', 'BERTRANDT', 'CAPGEMINI', 'CARREVOLUTIS', 'EIFFAGE', 'ELCIA', 'ELIS', 'JOBTEASER', 'LOREAL', 'MBDA', 'OPE', 'P&G','SAFEWARE','SEB','SNEF','SOLUTEC' ] %}
```
Comment (mehdibaha) [inline app/templates/users/forum-rhone-alpes.html]: there is a reason why filenames are in lowercase... some images aren't working:

image

File: app/templates/users/master-class.html (hunk 2 above)
Comment (momeg) [reply app/templates/users/master-class.html]: It was actually

Comment (mehdibaha) [reply app/templates/users/master-class.html]: ok

File: app/users/views.py (hunk 7 above)
Comment (momeg) [reply app/users/views.py]: inline comments

Comment (momeg) [reply app/users/views.py]: yep

Comment (mehdibaha) [reply app/users/views.py]: Inline comments are comments on the *same line* as a statement. Sorry, but there's no statement on line 98/99.

Actually, if you want to document a method, the (proper way is this)[https://www.python.org/dev/peps/pep-0008/#id33] (but you can just remove the two spaces before the method, it's ok...)

File: app/templates/users/dashboard/sections/dashboard.html (hunk 1 above)
Comment (momeg) [reply app/templates/users/dashboard/sections/dashboard.html]: If I do this clicking the link will result in opening the file explorer to upload the CV .

File: app/templates/users/dashboard/sections/profile.html [hunk 9]
```diff
@@ -121,10 +121,14 @@ <h3 class="box-title">
...
             <form id="dropzone" style="margin-bottom:0px" class="dropzone dz-clickable" method="POST" enctype="multipart/form-data">
               <div class="dz-message">
-                Glisse ton CV ou clique ici pour l'uploader.<br>
+                Glisse ton CV ou clique ici pour l'uploader.<br/>Et tente de gagner un MacBook Air le jour du salon. *
                 <span class="note">(Seuls les documents PDF sont acceptés.)</span>
               </div>
+
             </form>
+              <div class="dz-message">
+                  <a href="../../../../static/Reglement.pdf" target="_blank">* Modalités du concours</a>
```
Comment (mehdibaha) [inline app/templates/users/dashboard/sections/profile.html]: This is really really bad. Always use `url_for('static', ...)`

**Pull request closed without merging**